lib/apiservers/engine/backends/portmap/portmap.go

```diff
@@ -165,7 +177,7 @@ func (p *portMapper) forward(action iptables.Action, ip net.IP, port int, proto,
 		}
 		return fmt.Errorf("Failed to find unmap data for %s:%d", ipStr, port)
 
-	case iptables.Append:
+	case Map:
```

Comment:
Maybe a comment before the function saying we don't lock here because we lock in the calling function?